frontend/test/metabase/scenarios/onboarding/setup/user_settings.cy.spec.js

```diff
@@ -92,6 +92,23 @@ describe("user > settings", () => {
     cy.findByText("Sign in to Metabase");
   });
 
+  it.skip("should be able to change a language (metabase#22192)", () => {
```

Comment:
Should we specify `to Indonesian`? Because it seems to be the only one that doesn't work

Reply:
Ideally, we should expand this to all supported languages later on to avoid similar bugs. I'm ok with this title for now. There's an issue reference in the title whcih points out to Indonesian.

Reply:
That said, I don't have a strong preference. We can add "Indonesian" to the title if you think it makes it much more clear?